src/commands/executeFunction/executeFunction.ts

```diff
@@ -113,4 +141,7 @@ export async function executeFunction(context: IActionContext, node?: FunctionTr
 
     const message: string = responseText ? localize('executedWithResponse', 'Executed function "{0}". Response: "{1}"', func.name, responseText) : localize('executed', 'Executed function "{0}"', func.name);
     void window.showInformationMessage(message);
+
+    ext.isExecutingFunction = false;
```

Comment:
This whole function needs to be wrapped with a `try/finally`. If there's ever an error when executing the function, I believe that these extension values will not get reset.

Reply:
Done